android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java

```diff
@@ -49,6 +54,25 @@ public void removeTag(String tag) {
     }
 
     @ReactMethod
+    public void setQuietTime(ReadableMap time) {
+      try {
```

Comment:
More white spacing issues in this method. 4 spaces for indents. Everything in the try needs another 4 spaces.